src/components/Sprites/Sprites.tsx

```diff
@@ -2,69 +2,49 @@ import React from 'react'
 import './Sprites.scss'
 
 export const Sprites = ({ item }): JSX.Element => {
-  function shareOnVK(item: string) {
-    const navUrl = `http://vk.com/share.php?url=${window.location.href}&title=Today is ${item}`
-    window.open(navUrl, '_blank')
-  }
-
-  function shareOnTelegram(item: string) {
-    const navUrl = `https://t.me/share/url?url=${
-      window.location.href
-    }&text=Today is ${encodeURIComponent(item)}`
-    window.open(navUrl, '_blank')
-  }
-
-  function shareOnViber(item: string) {
-    const navUrl = `viber://forward?text=Today is ${item}`
-    window.open(navUrl, '_blank')
-  }
-
-  function shareOnWhatsApp(item: string) {
-    const navUrl = `whatsapp://send?text=Today is ${item}`
-    window.open(navUrl, '_blank')
-  }
-
-  function shareOnEmail(item: string) {
-    const navUrl = `mailto:?subject=Today is ${item}`
-    window.open(navUrl, '_blank')
-  }
+  
+  const vkUrl = `http://vk.com/share.php?url=${window.location.href}&title=Today is ${item}`
+  const telegramUrl = `https://t.me/share/url?url=${window.location.href}&text=Today is ${encodeURIComponent(item)}`
+  const viberUrl = `viber://forward?text=Today is ${item}`
+  const whatsappUrl = `whatsapp://send?text=Today is ${item}`
+  const emailUrl = `mailto:?subject=Today is ${item}`
+  const smsUrl = `sms:?body=Today is ${item}`
```

Comment:
you also can make it like an object instead of bunch of constants, for make it more readable